src/com/google/enterprise/adaptor/DocIdPusher.java

```diff
@@ -31,7 +31,18 @@
   public static final boolean EVERYTHING_CASE_SENSITIVE = true;
   public static final boolean EVERYTHING_CASE_INSENSITIVE = false;
 
-  /** FeedType used for group definition pushes. */
+  /**
+   * FeedType used for group definition pushes.
+   *
+   * GSA 7.4.0 added support for full feeds of group definitions. A {@code FULL}
+   * feed replaces all groups from a given data source. An {@code INCREMENTAL}
+   * feed augments or updates existing group definitions for a given data source.
+   * When feeding GSAs earlier than 7.4.0, {@code INCREMENTAL} feeds are always
+   * used. {@code INCREMENTAL} is the default FeedType for the overloaded
+   * {@code pushGroupDefinitions} methods that do specify a FeedType.
+   *
+   * @since 4.1.4
```

Comment:
I think @since TODO(bmj) would be better. We had a habit with v3 of looking for "@since TODO" before each release. You could keep this and add the other in an implementation comment just after this. /* @since TODO(bmj) */